src/server/routes/api-routes.js

```diff
@@ -5,4 +5,24 @@ const router = express.Router();
 
 router.use('/items', items);
 
+router.use('/*', (req, res) => {
+  if (res.statusCode === 200 || res.statusCode === 304) {
+    return;
+  }
+  res.status(404).send('API Request > 404 - Page Not Found');
+});
+
+router.use((errHandler, req, res) => {
+  res.status(errHandler.statusCode).send({
+    error: errHandler.message
+  });
+});
```

Comment:
Yes you are right.
But we dont use 4th arg , next
Unused variable is recognized Error in ESlint
So i remove it